sale_delivery_address_modify/models/sale_order.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Can you please use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py